webapp/client/src/lib/faqAnchors.js

```diff
@@ -61,11 +65,13 @@ const faqAnchorList = [
       <div>
         <p>{t("LandingPage.Accordion.Item2.detail")}</p>
         <ul>
-          <li>{trans("LandingPage.Accordion.Item2.listItem1")}</li>
-          <li>{trans("LandingPage.Accordion.Item2.listItem2")}</li>
+          <li>{t("LandingPage.Accordion.Item2.listItem1")}</li>
```

Comment:
are you sure about that? the trans() is used to replace Urls in the localisation

Reply:
yes, design changed the contents of this drop down. No inner urls except in the last p tag. So no need for the trans.